frame/assets/src/mock.rs

```diff
@@ -101,44 +101,53 @@ impl Config for Test {
 	type Extra = ();
 }
 
-use std::{cell::RefCell, collections::HashMap};
+use std::collections::HashMap;
 
 #[derive(Copy, Clone, Eq, PartialEq, Debug)]
-pub(crate) enum Hook {
+pub enum Hook {
 	Died(u32, u64),
 }
-thread_local! {
-	static FROZEN: RefCell<HashMap<(u32, u64), u64>> = RefCell::new(Default::default());
-	static HOOKS: RefCell<Vec<Hook>> = RefCell::new(Default::default());
+parameter_types! {
+	static FROZEN: HashMap<(u32, u64), u64> = Default::default();
+	static HOOKS: Vec<Hook> = Default::default();
```

Comment:
```suggestion
	static Frozen: HashMap<(u32, u64), u64> = Default::default();
	static Hooks: Vec<Hook> = Default::default();
```

The macro automatically converts them to SCREAM_CASE under the hood.